cmd/promtool/main.go

```diff
@@ -62,6 +62,15 @@ func CheckConfigCmd(t cli.Term, args ...string) int {
 	return 0
 }
 
+func checkFile(fn string) error {
```

Comment:
Could we rename this to `checkFileExists` now for clarity? Since it's no longer locally defined in a function right before its usage...